features/admin.users.v1/constants/user-management-constants.ts

```diff
@@ -464,3 +465,29 @@ export const RECOVERY_SCENARIO_TO_RECOVERY_OPTION_TYPE_MAP: Record<string, strin
 export enum AttributeDataType {
     COMPLEX = "COMPLEX"
 }
+
+/**
+ * User account status options for filtering users.
+ */
+export const USER_ACCOUNT_STATUS_FILTER_OPTIONS: DropdownChild[] = [
+    {
+        key: "LOCKED",
+        text: "Locked",
+        value: "urn:scim:wso2:schema:accountState eq 'LOCKED'"
```

Comment:
Add the key name

Reply:
https://github.com/wso2/identity-apps/pull/8171/commits/a35f3f308db99f0f624552fce3446dbdefa8b471